README.md

```diff
@@ -77,4 +89,4 @@ This repository contains helper scripts and commands to create meetup videos for
 
 * create the uploadable version
 
-        $ ffmpeg -i title.mp4 -i final2.mp4 -filter_complex "[0:0] [0:1] [1:0] [1:1] concat=n=2:v=1:a=1 [v] [a]" -map "[v]" -map "[a]" uploadable.mp4
+        $ ffmpeg -i title.mp4 -i final2.mp4 -filter_complex "[0:0] [0:1] [1:0] [1:1] concat=n=2:v=1:a=1 [v] [a]" -map "[v]" -map "[a]" -preset ultrafast uploadable.mp4
```

Comment:
Ultrafast probably is making a wager to bring the cores down to that percent. Not using it will take time and more resources but it'll bring down the final size of the video. 

-1 for having the `-present ultraflag` in here from me